x/events/events.go

```diff
@@ -45,39 +46,40 @@ const (
 	WebhookSucceeded         semconv.Event = "WebhookSucceeded"
 	WebhookFailed            semconv.Event = "WebhookFailed"
 	JsonnetMappingFailed     semconv.Event = "JsonnetMappingFailed"
+	LoginStarted             semconv.Event = "LoginStarted"
```

Comment:
We have RecoveryInitiated and no started yet, should we also call this LoginInitiated to reduce the number of individual namings?